index.html

```diff
@@ -120,7 +130,128 @@ <h2>Introduction</h2>
       <p class="issue" data-number="7"></p>
     </section>
 
-  <section id="conformance"></section>
+    <section id="conventions">
+      <h2>Documentation Conventions</h2>
+      <p>
+        This document uses the same conventions as [[TTML2]] for the specification of parameter attributes, styling attributes and metadata elements. In particular:
+      </p>
+      <p>
+        Section 2.3 of [[TTML2]] specifies conventions used in the [[XML]] representation of elements; and
+        Sections 6.2 and 8.2 of [[TTML2]] specify conventions used when specifying the syntax of attribute values.
+      </p>
```

Comment:
The TTML2 conventions are large and not all needed. I'm worried about making this spec unnecessarily hard to read. Can we rather pull in only the conventions we need? 
For example the "may"/"should"/must" part is already covered in the conformance section. Another example is the use of "deprecated"/"obsolete" that are not (yet) needed.

Reply:
I think it is clear enough that only a subset of the conventions are being brought in.

Reply:
It is not clear to me.